honeybee_radiance/cli/multiphase.py

```diff
@@ -510,7 +509,7 @@ def _aperture_view_factor(project_folder, apertures, size=0.2, ambient_division=
         sys.exit(0)
 
 
-@multi_phase.command('octrees-grids')
+@multi_phase.command('prepare-dynamic')
```

Comment:
I like the new name better but this is a breaking change. Am I right that we are currently only using this command in the multiphase recipes? If that's the case, then I'm fine with it, otherwise, we should probably don't change the name as it can break other recipes that are shipped under the lbt-recipes.

Reply:
It is only used in three-phase, so it will not break anything.